lemmy_api/src/lib.rs

```diff
@@ -292,6 +324,15 @@ pub async fn match_websocket_operation(
     UserOperation::CreateCommentLike => {
       do_websocket_operation::<CreateCommentLike>(context, id, op, data).await
     }
+    UserOperation::CreateCommentReport => {
+      do_websocket_operation::<CreateCommentReport>(context, id, op, data).await
+    }
+    UserOperation::ListCommentReports => {
+      do_websocket_operation::<ListCommentReports>(context, id, op, data).await
+    }
+    UserOperation::ResolveCommentReport => {
+      do_websocket_operation::<ResolveCommentReport>(context, id, op, data).await
+    }
   }
```

Comment:
Unrelated to this PR, but we should really rethink this `UserOperation` enum. Handling API calls by branching on an enum with 70 values is just horrible. Especially because this function is almost entirely boilerplate.

Reply:
What would be another way to do?

Reply:
I dont know. Maybe turn it into some kind of hierachy, so things are separeted into post actions, comment actions, report actions and so on?